docs/howto/basic-napalm-getters.rst

```diff
@@ -8,13 +8,13 @@ Let's start by seeing how to work with the inventory. Let's assume the following
 
 * Hosts file:
 
-.. literalinclude:: ../../examples/hosts.yaml
+.. literalinclude:: ../../examples/inventory/hosts.yaml
    :name: hosts.yaml
    :language: yaml
 
 * Groups file:
 
-.. literalinclude:: ../../examples/groups.yaml
+.. literalinclude:: ../../examples/inventory/groups.yaml
```

Comment:
I am going to add general comments about the tutorial here:

0. On line 57 it needs to be:
```
host["brigade_nos"]
```

1. The inconsistency between attribute names and keys is a bit strange? Should these be identical?

```
# dir(host) with underscore attributes filtered out
 'brigade',
 'connections',
 'data',
 'get',
 'get_connection',
 'group',
 'host',
 'items',
 'keys',
 'name',
 'network_api_port',
 'nos',
 'os',
 'password',
 'ssh_port',
 'username',
 'values'

In [9]: host.keys()
Out[9]: dict_keys(['name', 'group', 'domain', 'brigade_host', 'brigade_username', 'brigade_password', 'brigade_nos', 'brigade_port', 'site', 'role'])
```

The part that is a bit strange is that we have names that are similar, but not identical. Should we just make them identical. Also should we have a one-to-one correspondence between attribute access and dictionary keys (i.e. all dictionary keys could just be accessed as `host.foo`)?

2. Use of `brigade` as a variable name here:

```
     >>> brigade = Brigade(
     ...         inventory=SimpleInventory("hosts.yaml", "groups.yaml"),
     ...         dry_run=True)
```

This doesn't work:

```
>>> result = cmh_leaf.run(task=tasks.napalm_get_facts,
...                       facts="facts")
```

Instead needed to use:

```
from brigade.plugins.tasks.networking import napalm_get
result = cmh_leaf.run(task=napalm_get, getters='get_facts')
```

3. print(result) on line 88 doesn't work as shown:

```
# Here is what I saw at that point:
>>> print(result)
AggregatedResult: napalm_get
```

Here is what I ended up doing to print the result:

```
>>> for k, v in result.items():
...   print("-" * 80)
...   print(k)
...   for e in v:
...     print(e.result)
...   print("-" * 80)
```

4. I thought it was a bit strange in the output printing above that the result.items() that the `v` was a MultiResult and I needed to loop over it. The part that was strange to me was when we do the `get_task(task)` later in this tutorial...in that situation we just get a result that we can print and not a MultiResult (or vice versa). In other words, it isn't clear to me why we return a MultiResult in one context and not in the other.

5. There were some changes that I needed to make to get the `get_info(task)` to run properly. Once again for the import of `napalm_get` and for the result printing:

```
def get_info(task):
    # Grouping multiple tasks that go together
    r = napalm_get(task, "get_facts")
    print(task.host.name)
    print("============")
    print(r.result)
    r = napalm_get(task, "get_interfaces")
    print(task.host.name)
    print("============")
    print(r.result)
```

Reply:
Okay...maybe this is all old as it is in `docs/howto` and not `docs/tutorials`. Probably should only have one place and not three (`examples`, `docs/howto`, `docs/tutorials` and I am 0 for 2 on reviewing the right one).

Reply:
It is due to the structure of the documentation in general. Saw that you were talking about this on Twitter too. Basically howto's aim to solve a specific task and tutorials is more handholding. Currently they might basically cover the same topics. Doesn't look like the howtos are included in this PR, but they probably need to be updated.

Also think that the examples are good for people who only want to view the code and not have it spreadout on a page with different sections as can happen in a tutorial.